core/iwasm/interpreter/wasm.h

```diff
@@ -10,6 +10,10 @@
 #include "bh_hashmap.h"
 #include "bh_assert.h"
 
+#if WASM_ENABLE_LOAD_CUSTOM_SECTION != 0
+#include "wasm_export.h"
```

Comment:
wasm_export.h is included in wasm_runtime_common.h, how about not including wasm_export.h in this file?
And try using "struct wasm_section_t *custom_section_list" below?